packages/grpcts/src/Service.ts

```diff
@@ -56,23 +57,23 @@ const logging = (
   logger: Logger,
   definition: grpc.MethodDefinition<any, any>,
   call: any,
-  result?: any
+  result?: any,
+  latency?: ILatency
 ) => {
-  const logData: {
-    request?: any;
-    response?: any;
-    error?: any;
-    path: string;
-  } = {
-    path: definition.path
+  const request = !definition.requestStream ? call.request : 'STREAM';
+  const response = !definition.responseStream ? result : 'STREAM';
+  const isError = result instanceof Error;
+  const logData = {
+    request,
+    [isError ? 'error' : 'response']: response,
+    path: definition.path,
+    emitter: 'service'
```

Comment:
```suggestion
    emitter: 'service',
    latency: latency?.getValue()
```
Would this work?

Reply:
true. it was optional from the beginning,  but i added default timer